### ISSUE TITLE
Add the language a contact/case/event participant speaks 

### ISSUE THREAD

**Issue by markusmann-vg** [SORMAS-Foundation/SORMAS-Project#2858]
### Situation Description
Currently there is no such field that describes, which language(s) a contact/case/event participant speaks. 
That should be added to the forms and exports

### Feature Description
That should be added to the forms and exports. A list will be provided by @GabrielePoggensee 
The field should be a dropdown with multiple selection enabled

**Comment by ChristopherRiedel** [comment]
Doesn't such a field rather belong to a person?

**Comment by Candice-Louw** [comment]
Request for CH:
en | de-CH
|--|--|
German | Deutsch
French | Französisch
Italian | Italienisch
Romansh | Rätoromanisch
Other | Andere

If `Other | Andere` > Free text field to enter value

Makes sense to add it to the person entity, yes. @bernardsilenou - Would this be something of interest for other countries too, or do we keep this CH-specific only?

**Comment by bernardsilenou** [comment]
@Candice-Louw Yes please, it is good to add to the person entity. It will be good to be configurable based on the country. This list makes sense for a Swiss server but no for an African server. 
@markusmann-vg  If this will require much resources for now, then we can add to the person entity on Swiss server only.

**Comment by markusmann-vg** [comment]
**Swiss server only:**
- [ ] Add country-dependent enum to person, which can be added and edited in a popup 
- [ ] Reuse the functionality of statistics, where you can add more filters

**Comment by CriSi315** [comment]
+upvote.
In Berlin, we have many cases / contact persons who don't speak German.

It is vital for the contact tracers to have a field to record in which language a person can be contacted!

**Comment by JaquM** [comment]
I would add the same for german server with following languages:

|en|DE|
|-|-|
|German|Deutsch|
Russian|Russisch|
|Turkish|Türkisch|
|Arabic|Arabisch|
|Polish|Polnisch|
|Kurdish|Kurdisch|
|English|Englisch|
|Romanian|Rumänisch|
|Spanish|Spanisch|
|Italian|Italienisch|
|Greek|Griechisch|
|Other|Andere|

**Comment by bernardsilenou** [comment]
We can use the same logic we used for disease variants, thus permitting each instance to define the languages

**Comment by Candice-Louw** [comment]
@MateStrysewske - would @bernardsilenou 's solution be fine from your perspective?

Alternatively, perhaps uploading `Languages` as `Configuration Data` may be an option?

**Comment by JonasCir** [comment]
How nicely do instance defined data (language/disease variants etc.) play with S2S? Isn't this the same as infrastructure data?

**Comment by Candice-Louw** [comment]
@JonasCir - you're right - not a good combination for S2S.  

Question - with the new interface [Angular], would it be possible to:
1. Provide a list of all languages in the world to have listed in SORMAS DB (for all instances) as defined (ISO list for example)
2. Allow toggling which languages should be visible, per instance, by configuration of `admin config users` on the `admin dashboard` (https://github.com/hzi-braunschweig/SORMAS-Project/issues/4410) on their own instance?
3. Once data is sent S2S, it would be able to be saved in the DB of the new instance.  It would make saving possible at least.

Feedback, please?

**Comment by MateStrysewske** [comment]
@bernardsilenou @Candice-Louw This issue needs internal technical refinement, that's why the `refine` label has been added. I don't think there's anything you need to do about it right now. We might need a similar approach to the countries, but one of the problems is to determine which language list to choose, and how extensive it should be (because there are, of course, a lot of languages).
